src/promptflow/promptflow/_sdk/_serving/response_creator.py

```diff
@@ -15,7 +15,12 @@ class ResponseCreator:
     """Generates http response from flow run result."""
 
     def __init__(
-        self, flow_run_result, accept_mimetypes, stream_start_callback_func=None, stream_end_callback_func=None
+        self,
+        flow_run_result,
+        accept_mimetypes,
+        stream_start_callback_func=None,
+        stream_end_callback_func=None,
+        stream_event_callback_func=None
```

Comment:
add test for the callback_func?

Reply:
this maybe not required, this is just adding a new event hook and there's no real biz logic in this.